### PR TITLE
Add dropout in BaseNN introduction

### PR DESCRIPTION
**Pull request by bacz00**
之前尝试添加dropout层，在layer里输入'dropout’后看到`增加dropout层,参数置零的概率为: 0.5`，但查阅文档发现没有关于添加dropout层，设置概率的描述，于是去查看源代码
```
elif layer == 'dropout':
    p = 0.5 if 'p' not in kw.keys() else kw['p']
    self.model.add_module('dropout' + str(self.layers_num), torch.nn.Dropout(p=p) )
    print("增加dropout层,参数置零的概率为: {} ".format(p))
```
看到通过p参数设置概率。

个人认为写入文档对使用者更加便利。
如果有错误，希望能够指出，谢谢！

### CODE REVIEW

**Comment by EasonQYS** [comment]
thanks, we now add this info in our docs.
